vault/cluster.go

```diff
@@ -352,7 +364,7 @@ func (c *Core) ClusterTLSConfig(ctx context.Context, repClusters *ReplicatedClus
 			localCert := make([]byte, len(currCert))
 			copy(localCert, currCert)
 
-			//c.logger.Trace("core: performing cert name lookup", "hello_server_name", clientHello.ServerName, "local_cluster_cert_name", parsedCert.Subject.CommonName)
+			//c.logger.Trace("performing cert name lookup", "hello_server_name", clientHello.ServerName, "local_cluster_cert_name", parsedCert.Subject.CommonName)
```

Comment:
Remove or enable commented out code.

Reply:
@jefferai is it cool if I strip out that line or would it be useful for me to keep it like that?

Reply:
I guess strip out, haven't needed that one in a while.